ccm_web/server/src/canvas/canvas.scopes.ts

```diff
@@ -8,7 +8,9 @@ can be found in the API Developer Key interface in Canvas.
 const privilegeLevelOneScopes = [
   // Courses
   'url:GET|/api/v1/courses/:id',
-  'url:PUT|/api/v1/courses/:id'
+  'url:PUT|/api/v1/courses/:id',
+  // Sections
+  'url:GET|/api/v1/courses/:course_id/sections'
```

Comment:
I wasn't very worried about this, because it was working for me without it (because my key is unrestricted) and I saw it has been added to the section creation API function PR.  But I'll accept it anyway. 🙂